src/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import combineReducers from './redux/configureStore';
 import App from './App';
 
+const store = configureStore({ combineReducers });
+
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
```

Comment:
- [ ] I kindly suggest that you make sure that your Bookstore app is running because I can only see a blank white space when I run `npm run start`

https://www.loom.com/share/3b6b698949da4d9ea0f6e7698df6a0b2